Jenkinsfile.k8s

```diff
@@ -78,7 +78,14 @@ pipeline {
             export NP=`grep -c ^processor /proc/cpuinfo`
             mkdir -p /home/dealii/build-gcc-fast
             cd /home/dealii/build-gcc-fast
+
+            # We need to add this option to mpirun, otherwise
+            # openmpi complains about running as root. We need
+            # to run as root, because we are inside a docker container,
+            # and the surrounding containers assume we have root access.
```

Comment:
I'm pretty sure it's the Jenkins/Kubernetes interaction that requires running as root, and not the fact that it's a Docker container.